frontend/app/src/utils/date.ts

```diff
@@ -24,39 +24,44 @@ export function changeDateFormat(
     return '';
   }
 
-  const seconds = date.charAt(date.length - 6) === ':';
   const timestamp = convertToTimestamp(date, fromFormat);
 
-  return convertFromTimestamp(timestamp, seconds, toFormat);
+  return convertFromTimestamp(timestamp, toFormat);
 }
 
 export function convertToTimestamp(
   date: string,
   dateFormat: DateFormat = DateFormat.DateMonthYearHourMinuteSecond
 ): number {
   let format: string = getDateInputISOFormat(dateFormat);
-  if (date.includes(' ')) {
+  const firstSplit = date.split(' ');
+  if (firstSplit.length === 2) {
     format += ' HH:mm';
-    if (date.charAt(date.length - 6) === ':') {
+
+    const secondSplit = firstSplit[1].split(':');
+    if (secondSplit.length === 3) {
       format += ':ss';
+
+      const thirdSplit = secondSplit[2].split('.');
+      if (thirdSplit.length === 2) {
+        format += '.SSS';
+      }
     }
   }
 
-  return dayjs(date, format).unix();
+  return dayjs(date, format).valueOf() / 1000;
```

Comment:
`unix()` returns the epoch in seconds 
`valueOf()` returns the epoch in ms

1 second is 1000 ms so if you do valueOf()/1000 you get the epoch in seconds, which is what unix does so why change it?